app/views/root/homepage.html.erb

```diff
@@ -1,13 +0,0 @@
-<%
- # There is a requirement to add the thank you comment in a place which is
- # obvious. Rendering file with the govuk_template lets us put the comment
- # after the govuk_template. govuk_template as it stands doesn't have a
- # content_for which could otherwise be used.
-
- # This is only wanted for the homepage and can't be put in to the frontend
- # template on frontend due to the way slimmer would not copy over the
- # comment
-%>
-<%= render partial: 'base', locals: { hide_nav: true } %>
-<%= render template: 'layouts/govuk_template' %>
-<!-- Thanks Martha -->
```

Comment:
I had a quick look to see where this comment comes out now but I can't see it - do we no longer have this? Was there a decision made to remove it from GOV.UK entirely?

Reply:
That is a good point, Andy. I will move it in the `core-layout` template in this PR so we don't forget about it.

Reply:
I just checked and I can't see it on the live homepage 😱